### ISSUE TITLE
Ability to add comments/detailed explanation, if needed, in lists of instances when an item is flagged as a failure.

### ISSUE THREAD

**Issue by anipv** [microsoft/accessibility-insights-web#7017]

**Comment by DaveTryon** [comment]
Hi, @anipv! Thanks for your feature suggestion. We've reviewed it and added it to our internal feature backlog. It's impossible to predict when this might make it to the web extension, but we'll do our best to make it happen. Thanks for using Accessibility Insights!

**Comment by anipv** [comment]
Hi there!
Thanks for the follow up reply. I see there's another similar "Need Help" request under Issues [(https://github.com/microsoft/accessibility-insights-web/issues/485)]((https://github.com/microsoft/accessibility-insights-web/issues/485)). 
How can I help to make this happen sooner :-)
TIA

**Comment by DaveTryon** [comment]
@anipv You're correct that this is basically a dupe of #485. We had a huge backlog that we were never going to be able to complete, so this got removed as part of the backlog cleanup. I don't have a great answer on how to move this along, other than to contribute a PR that add the code in all the places it needs to be (UX, reading and writing this in both Assessment and QuickAssess, tests, etc.) It's a fairly large piece of work, which is probably why it kept getting pushed aside by other things. We have some other large items in our backlog that will realistically take some time to complete, but at least this is back on the radar.

Thanks for using Accessibility Insights!
